code/items/equipments/entities/C4Entity.cs

```diff
@@ -12,28 +13,62 @@ namespace TTTReborn.Items
     [Library("ttt_c4_ent"), Hammer.Skip]
     public partial class C4Entity : Prop, IUse
     {
+        public static C4Preset[] TimerPresets =
+        {
+            new C4Preset
+            {
+                timer = 60,
+                wires = 1
+            },
+
+            new C4Preset
+            {
+                timer = 150,
+                wires = 2
+            },
+
+            new C4Preset
+            {
+                timer = 300,
+                wires = 4
+            }
+        };
+
         private string ModelPath => "models/entities/c4.vmdl";
 
         [Net]
         public int AttachedBone { get; set; } = -1; //Defaults to -1, which indicates no bone attached as this value will not always be set.
 
         [Net]
-        public bool IsArmed { get; set; } = false;
+        public bool IsArmed
```

Comment:
Is this needed? Could just use the enum, I guess it does generally increase readability so I'll leave it up for you to decide.

Reply:
Yeah, I wasn't entirely sure if this was needed. Not sure if it needs to be marked [Net] either if the State is marked [Net] already.

Reply:
If State is networked, should be no reason to network it as well if it's getter just receives the networked value. It'd pull the current value for whatever realm it's running in anyways.